templates/web-application/.storybook/main.ts

```diff
@@ -0,0 +1,55 @@
+import type { StorybookConfig } from "@storybook/react-webpack5";
+import type  { Configuration } from "webpack";
+
+const config: StorybookConfig = {
+  stories: [
+    "../src/**/*.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "storybook-addon-swc",
+  ],
+  framework: {
+    name: "@storybook/react-webpack5",
+    options: {},
```

Comment:
Is the empty `options` object required?

Reply:
options is required.

Reply:
Property 'options' is missing in type '{ name: "@storybook/react-webpack5"; }'